vendors/circom/circomlib/base/g1_affine_point.h

```diff
@@ -24,19 +24,19 @@ struct G1AffinePoint {
     return !operator==(other);
   }
 
-  template <typename Curve>
+  template <bool Motgomery, typename Curve>
```

Comment:
Please fix typo `Motgomery` -> `Montgomery`.
Also, how about using `IsMontgomery` instead for clarity?